.github/workflows/test.yml

```diff
@@ -0,0 +1,35 @@
+name: Test colcon-ros-bundle
+on:
+  pull_request:
+  schedule:
+    - cron: '43 17 * * *'
```

Comment:
Can you add a comment explaining what this means in English?

Reply:
Done